src/config.ts

```diff
@@ -146,6 +147,26 @@ export class KubeConfig implements SecurityAuthentication {
         this.makePathsAbsolute(rootDirectory);
     }
 
+    public async applytoFetchOptions(opts: https.RequestOptions): Promise<RequestInit> {
```

Comment:
could we just merge this with applyToHTTPSOptions()?

Reply:
We can not, as `applyToHTTPSOptions` is being used by proto-client